hack/update-rhcos-bootimage.py

```diff
@@ -8,10 +8,11 @@
 # builds.  Do not try to e.g. point to RHT-internal endpoints.
 RHCOS_RELEASES_APP = 'https://releases-art-rhcos.svc.ci.openshift.org'
 
-dn = os.path.abspath(os.path.dirname(sys.argv[0]))
+RHCOS_BUILD_METADATA = os.path.join(os.path.dirname(sys.argv[0]), "../data/data/rhcos.json")
 
 parser = argparse.ArgumentParser()
 parser.add_argument("meta", action='store')
+parser.add_argument("arch", action='store', choices=['amd64'])
```

Comment:
That's the golang/Debian name for the arch; for RHCOS at least we use `x86_64` which is the RPM name.

In practice we often need to [translate between them](https://github.com/coreos/coreos-assembler/blob/56952ffae17baf542f13563a2a4ae0e45ffeccfb/Makefile#L23) - not saying that choosing `amd64` here is wrong exactly, but I think there's a stronger argument for matching what RHCOS uses.

On the other hand, manifest listed OCI images do use `amd64` - so we'll need to juggle both.

Reply:
Related: https://github.com/openshift/installer/pull/2885/files#r363720859

Reply:
I mentioned above that this value is directly used in the install config, so I opted to go with what we already expose to our customers. Since this `x86_64`/`amd64` "mismatch" happens behind the scenes, I'm inclined to leave it as is.